### PR TITLE
Misc. typo fixes in comments and documentation

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -w -L ith,als,dum,nd,amin,nto,wis,ba -S ./relapack,./kernel,./lapack-netlib`

### CODE REVIEW

**Comment by luzpaz** [comment]
JFYI, fixes to upstream https://github.com/HPAC/ReLAPACK/pull/9 have been merged as well (found during this PR).

File: driver/others/blas_server.c [hunk 1]
```diff
@@ -150,8 +150,8 @@ static unsigned int thread_timeout = (1U << (THREAD_TIMEOUT));
 
 #ifdef MONITOR
 
-/* Monitor is a function to see thread's status for every seconds. */
-/* Usually it turns off and it's for debugging.                    */
+/* Monitor is a function to see thread's status for every second. */
```
Comment (luzpaz) [inline driver/others/blas_server.c:153]: please review

**Comment by martin-frbg** [comment]
Thanks. (I changed the title to make it clearer that this commit will have no effect on actual code). FYI "reference" is actually dead code, some of the TODOs and comments can probably be removed completely.

**Comment by luzpaz** [comment]
Thanks.   
@martin-frbg how do you want me to proceed? Revert hte reference/ commits ?

**Comment by brada4** [comment]
`reference/` is copy of old netlib blas, a component of test framework that never happened.

**Comment by luzpaz** [comment]
please advise

**Comment by brada4** [comment]
The advice is not to change files in that directory. It will get removed by some release when someone takes courage to clean ifdef-s scattered around refering to it.

**Comment by martin-frbg** [comment]
I guess this is predominantly a matter of personal taste - I just see little point in making lots of fairly static (or even dead) code suddenly appear changed just to do away with typos (or Japanese spelling glitches) that have been there for ten years minimum. So perhaps we can agree on leaving the unused "reference" unchanged at least until its final fate gets decided ?

**Comment by luzpaz** [comment]
Reverted `reference/` fixes in 858e609

**Comment by brada4** [comment]
Changelog also should keep the mistakes....

**Comment by luzpaz** [comment]
Done. b46875b

**Comment by martin-frbg** [comment]
Thanks (I'd actually have been happy with the Changelog fixes as that is definitely a "customer-facing" document :slightly_smiling_face: - I'll try to remember to rectify it with the next update)

**Pull request merged**